.eleventy.js

```diff
@@ -2,6 +2,17 @@ import govukEleventyPlugin from "@x-govuk/govuk-eleventy-plugin";
 
 export default function (eleventyConfig) {
 
+    // The name of the repo as it appears on GitHub
+    const repoName = '';
+
+    const url = process.env.GITHUB_ACTIONS
+        ? `https://ukhomeoffice.github.io/${repoName}/`
+        : '/';
```

Comment:
```suggestion
    /** TODO: this should be configured to match the url where this site is hosted. 
      * For example when using a GitHub action to deploy to GitHub pages:
      * 
      * ```javascript
      * const url = process.env.GITHUB_ACTIONS
      *    ? `https://ukhomeoffice.github.io/${repoName}/`
      *    : '/';
      */
    const url = '/';
```